src/devices/futurehome.ts

```diff
@@ -0,0 +1,96 @@
+import * as exposes from '../lib/exposes';
+import * as tuya from '../lib/tuya';
+
+const e = exposes.presets;
+const ea = exposes.access;
+
+const definition = {
```

Comment:
Please change this to https://github.com/Koenkk/zigbee-herdsman-converters/blob/d69dc7a13fbf9254219c5cfda84e50f5629bf1d0/src/devices/tuya.ts#L514

(this is what makes the build fail)

Reply:
I think this is resolved